homeassistant/components/evohome/__init__.py

```diff
@@ -674,8 +692,8 @@ def _dt_evo_to_aware(dt_naive: dt, utc_offset: timedelta) -> dt:
                 )
 
                 self._setpoints[f"{key}_sp_from"] = dt_aware.isoformat()
-                try:
-                    self._setpoints[f"{key}_sp_temp"] = switchpoint["heatSetpoint"]
+                try:  # heatSetpoint is the only pascalCase string
+                    self._setpoints[f"{key}_sp_temp"] = switchpoint[SZ_HEAT_SETPOINT]
                 except KeyError:
                     self._setpoints[f"{key}_sp_state"] = switchpoint["DhwState"]
```

Comment:
`DhwState` missed?

Reply:
No, not exactly.

The async client has an anachronism. It _should_ be 'dhwState`, but if I imported/used `SZ_DHW_STATE`, I'd get `DhwState`, and it would break.

You'll see quite a few others that are PascalCase, but should be camelCase.

In future, I will be changing all those literals to snake_case, and then can import all the corresponding constants. I haven't done so, because it will be a breaking change for other users of the client library.

But I thought this change would be a good first step.